we can't use col(1), because it was [gated behind if(__GLIBC__)][1]
in util-linux; according to upstream, it's also deprecated

[1]: https://github.com/util-linux/util-linux/commit/8886d84e25a457702b45194d69a47313f76dc6bc

diff --git a/xonsh/completers/man.py b/xonsh/completers/man.py
index a2e56f1..944ae6b 100644
--- a/xonsh/completers/man.py
+++ b/xonsh/completers/man.py
@@ -25,2 +25,3 @@ def get_man_completions_path() -> Path:
 def _get_man_page(cmd: str):
+    return None
     """without control characters"""
